frontend/src/pages/UserSettings/PlayerForm/PlayerForm.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+	INDEXEDDB_ENABLED_LOCAL_STORAGE_PREFIX,
+	indexeddbEnabled,
+} from '@util/db'
+import { Checkbox } from 'antd'
+import React from 'react'
+
+import styles from './PlayerForm.scss'
+
+export const PlayerForm = () => {
+	const [useIndexedDB, setUseIndexedDB] =
+		React.useState<boolean>(indexeddbEnabled)
+
+	return (
+		<>
+			<h3>App Settings</h3>
+			<form>
+				<p>
+					IndexedDB Browser Cache is used to preload all data in
+					highlight and speedup repeated loading, but can increase
```

Comment:
* "highlight" -> "Highlight"
* "speedup" -> "speed up"
* "but can increase" -> "but it can increase"

Reply:
thank you!